content/_components/rest-api-component.md

```diff
@@ -177,23 +203,24 @@ Rest-api component automatically load binary data to attachments with next conte
 * application/octet-stream
 
 ## Exception handling
-Rest API component uses exception handling logic below:
+Rest API component uses exception handling logic below: 
 ![Exception handling logic](https://user-images.githubusercontent.com/13310949/41960520-9bd468ca-79f8-11e8-83f4-d9b2096deb6d.png)
 
 ## Known Limitations
 
-The component can parse any of json and xml content types.
+The component can parse any of json and xml content types. 
 There are:
 * application/json
 * application/xml
 * text/xml
 * etc.
 
-`If content type is not  exists  in response header, component will try parse response as json.
+`If content type is not  exists  in response header, component will try parse response as json. 
 If it get parse exception, it return response as is.`
 
 > Make sure not to perform your tests using the [requestb.in](https://requestb.in/) since it responds with the `content-type: text/html`.
```

Comment:
Remove `https://requestb.in/`. This link is not working and the service is long one. Maybe completely remove this line or mention that `response type : text/html` would not be processed.